2019-03-20T21:41:51.991999+00:00 app[web.1]: (node:69) UnhandledPromiseRejectionWarning: error: relation "questionanswers" does not exist
2019-03-20T21:41:51.992001+00:00 app[web.1]:     at Connection.parseE (/app/backend/node_modules/pg/lib/connection.js:555:11)
2019-03-20T21:41:51.992002+00:00 app[web.1]:     at Connection.parseMessage (/app/backend/node_modules/pg/lib/connection.js:380:19)
2019-03-20T21:41:51.992003+00:00 app[web.1]:     at TLSSocket.<anonymous> (/app/backend/node_modules/pg/lib/connection.js:120:22)
2019-03-20T21:41:51.992005+00:00 app[web.1]:     at TLSSocket.emit (events.js:189:13)
2019-03-20T21:41:51.992006+00:00 app[web.1]:     at TLSSocket.EventEmitter.emit (domain.js:441:20)
2019-03-20T21:41:51.992007+00:00 app[web.1]:     at addChunk (_stream_readable.js:284:12)
2019-03-20T21:41:51.992008+00:00 app[web.1]:     at readableAddChunk (_stream_readable.js:265:11)
2019-03-20T21:41:51.992010+00:00 app[web.1]:     at TLSSocket.Readable.push (_stream_readable.js:220:10)
2019-03-20T21:41:51.992011+00:00 app[web.1]:     at TLSWrap.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
2019-03-20T21:41:51.992068+00:00 app[web.1]: (node:69) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
2019-03-20T21:41:51.992143+00:00 app[web.1]: (node:69) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
2019-03-20T21:41:52.013335+00:00 app[web.1]: (node:69) UnhandledPromiseRejectionWarning: error: relation "questionanswers" does not exist
2019-03-20T21:41:52.013338+00:00 app[web.1]:     at Connection.parseE (/app/backend/node_modules/pg/lib/connection.js:555:11)
2019-03-20T21:41:52.013339+00:00 app[web.1]:     at Connection.parseMessage (/app/backend/node_modules/pg/lib/connection.js:380:19)
2019-03-20T21:41:52.013341+00:00 app[web.1]:     at TLSSocket.<anonymous> (/app/backend/node_modules/pg/lib/connection.js:120:22)
2019-03-20T21:41:52.013342+00:00 app[web.1]:     at TLSSocket.emit (events.js:189:13)
2019-03-20T21:41:52.013343+00:00 app[web.1]:     at TLSSocket.EventEmitter.emit (domain.js:441:20)
2019-03-20T21:41:52.013347+00:00 app[web.1]:     at addChunk (_stream_readable.js:284:12)
2019-03-20T21:41:52.013349+00:00 app[web.1]:     at readableAddChunk (_stream_readable.js:265:11)
2019-03-20T21:41:52.013350+00:00 app[web.1]:     at TLSSocket.Readable.push (_stream_readable.js:220:10)
2019-03-20T21:41:52.013351+00:00 app[web.1]:     at TLSWrap.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
2019-03-20T21:41:52.013447+00:00 app[web.1]: (node:69) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
2019-03-20T21:41:52.042167+00:00 app[web.1]: (node:69) UnhandledPromiseRejectionWarning: error: relation "questionanswers" does not exist
2019-03-20T21:41:52.042171+00:00 app[web.1]:     at Connection.parseE (/app/backend/node_modules/pg/lib/connection.js:555:11)
2019-03-20T21:41:52.042172+00:00 app[web.1]:     at Connection.parseMessage (/app/backend/node_modules/pg/lib/connection.js:380:19)
2019-03-20T21:41:52.042174+00:00 app[web.1]:     at TLSSocket.<anonymous> (/app/backend/node_modules/pg/lib/connection.js:120:22)
2019-03-20T21:41:52.042176+00:00 app[web.1]:     at TLSSocket.emit (events.js:189:13)
2019-03-20T21:41:52.042178+00:00 app[web.1]:     at TLSSocket.EventEmitter.emit (domain.js:441:20)
2019-03-20T21:41:52.042180+00:00 app[web.1]:     at addChunk (_stream_readable.js:284:12)
2019-03-20T21:41:52.042182+00:00 app[web.1]:     at readableAddChunk (_stream_readable.js:265:11)
2019-03-20T21:41:52.042183+00:00 app[web.1]:     at TLSSocket.Readable.push (_stream_readable.js:220:10)
2019-03-20T21:41:52.042185+00:00 app[web.1]:     at TLSWrap.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
2019-03-20T21:41:52.042255+00:00 app[web.1]: (node:69) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)